hooks/submit_for_review.py

```diff
@@ -263,56 +268,57 @@ def publish(self, settings, item):
         :param item: Item to process
         """
 
-        sg_publish = item.properties["sg_publish_data"]
+        render_path = item.properties.get("path")
+        if render_path is None:
+            raise Exception("'path' property cannot be found in item's properties. "
+                            "Review Submission failed.")
+
+        sg_publish_data = item.properties.get("sg_publish_data")
+        if sg_publish_data is None:
+            raise Exception("'sg_publish_data' was not found in the item's properties. "
```

Comment:
Maybe add a line in here about this being expected to be populated by a publish plugin that runs before this.

Reply:
Added